packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -517,20 +519,22 @@ class _RadialPainter extends CustomPainter {
     for (_Circle circle in circles) {
       final Tween<double> radiusTween = new Tween<double>(
         begin: 0.0,
-        end: _maxRadius(circle.offset, size)
+        end: _maxRadius(circle.offset, size),
       );
       final Paint paint = new Paint()..color = circle.color;
       final Rect rect = new Rect.fromLTWH(0.0, 0.0, size.width, size.height);
       canvas.clipRect(rect);
       final double navWidth = math.min(bottomNavMaxWidth, size.width);
+      final double halfNavWidth = navWidth / 2.0;
+      final double halfHeight = size.height / 2.0;
       final Offset center = new Offset(
-        (size.width - navWidth) / 2.0 + circle.offset.dx * navWidth,
-        circle.offset.dy * size.height
+        (size.width - navWidth) / 2.0 + halfNavWidth + circle.offset.x * halfNavWidth,
```

Comment:
if you meant to change this, please make sure there's a mock_canvas test of the change

Reply:
Oh never mind, I see.